File: C/Colleen.c
#include <stdio.h>

/*
	first comment outside the main function
*/

void	print()
{
	char	*str = "#include <stdio.h>%1$c%1$c/*%1$c	first comment outside the main function%1$c*/%1$c%1$cvoid	print()%1$c{%1$c	char	*str = %3$c%2$s%3$c;%1$c	printf(str, 10, str, 34);%1$c}%1$c%1$cint	main()%1$c{%1$c	/*%1$c		second comment inside the main function%1$c	*/%1$c	print();%1$c	return (0);%1$c}%1$c";
	printf(str, 10, str, 34);
}

int	main()
{
	/*
		second comment inside the main function
	*/
	print();
	return (0);
}

File: PYTHON/Colleen.py

#!/usr/bin/python3

# first comment outside the main function

STRING = """
#!/usr/bin/python3

# first comment outside the main function

STRING = {q}{q}{q}{s}{q}{q}{q}

def print_program():
	print(STRING.format(q=chr(34), s=STRING), end="")

def main():
	# second comment inside the main function
	print_program()

if __name__ == "__main__":
	main()
"""

def print_program():
	print(STRING.format(q=chr(34), s=STRING), end="")

def main():
	# second comment inside the main function
	print_program()

if __name__ == "__main__":
	main()
